app/controllers/nodes_controller.rb

```diff
@@ -6,7 +6,7 @@ class NodesController < ApplicationController
 
   def show
 
-    @node = Node.published.find_by_path! params[:path] || ''
+    @node = Node.find_by_path! params[:path] || ''
```

Comment:
Won't this allow the viewing of any node publicly regardless of whether or not it has been published?